debugger/src/main/kotlin/org/rust/debugger/runconfig/RsDebugRunner.kt

```diff
@@ -167,7 +173,7 @@ private fun buildProjectAndGetBinaryArtifactPath(project: Project, command: Carg
                         }
 
                         val parser = JsonParser()
-                        result = output.stdoutLines
+                        result = output.stdoutLines.asSequence()
```

Comment:
Maybe `asSequence` doesn't need here? Looks like it's not a hot place in our code where we can spend a lot of time only for intermediate allocations but using sequences always make debugging significantly less convenient
Instead of this, we can collapse some sequential calls of `mapNotNull` and `filter`